frontend/src/scenes/urls.ts

```diff
@@ -209,6 +209,8 @@ export const urls = {
     // Cloud only
     organizationBilling: (products?: ProductKey[]): string =>
         `/organization/billing${products && products.length ? `?products=${products.join(',')}` : ''}`,
+    organizationBillingOverview: (): string => `/organization/billing/overview`,
```

Comment:
This pattern is good for now but we may want to move to how settings is done (look above in file) where it's a function that passes in the section id for the page.

Reply:
Moved into a function that receives a `BillingSectionId`